specification.json

```diff
@@ -294,6 +301,28 @@
             "RFC 2119 keyword": "MUST",
             "children": []
         },
+        {
+            "id": "Requirement 2.1.2",
+            "machine_id": "requirement_2_1_2",
+            "content": "The provider MAY define an `allFlagMetadata` field or accessor of type collection of [flag_metadata](../types.md",
```

Comment:
Not sure if it is a real problem, but the markdown link here is cut in half.

Reply:
Good catch @thomaspoignant. Markdown should be avoided in the requirements but work fine in the non-normative sections.